konsave/funcs.py

```diff
@@ -0,0 +1,276 @@
+## IMPORT ##
+import os, shutil, argparse, configparser
+from random import shuffle
+from zipfile import is_zipfile, ZipFile
+from konsave.vars import *
+
+from pkg_resources import resource_stream, resource_filename
+
+try:
+    import yaml
+except ModuleNotFoundError:
+    raise ModuleNotFoundError("Please install the module PyYAML using pip: \n"
+                              "pip install PyYAML")
+
+
+## FUNCTIONS ##
+def mkdir(path):
+    '''
+    Creates directory if it doesn't exist
+    '''
+    if not os.path.exists(path):
+        os.makedirs(path)
+    return path
+
+
+def load_config():
+    """
+    Load the yaml file which contains the files and folder to be saved
+
+    The file should be formatted like this:
+
+    ---
+    entries:
+        - folder name
+        - file name
+        - another file name
+        - another folder name
+
+    """
+    default_config_path = resource_filename('konsave', 'conf.yaml')
+    if not os.path.exists(CONFIG_FILE):
+        print_msg(f"No config file found! Using default config ({default_config_path}).")
+        shutil.copy(default_config_path, CONFIG_FILE)
+        print_msg(f"Saved default config to: {CONFIG_FILE}")
+        return yaml.load(resource_stream('konsave', 'conf.yaml'), Loader=yaml.FullLoader)["entries"]
```

Comment:
If the config file is not found the default config from the pip install folder will be copied to '~/.config/konsave'